pkgs/by-name/gi/git-worktree-switcher/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  installShellFiles,
+  git,
+  jq,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "git-worktree-switcher";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "mateusauler";
+    repo = "git-worktree-switcher";
+    rev = "refs/tags/${version}-fork";
+    hash = "sha256-fjY+ieUI7m/4gb68/1a+90aIc9iMyQortVd7oHN4STw=";
+  };
+
+  buildInputs = [
+    jq
+    git
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+    installShellFiles
+  ];
+
+  patches = [
+    ./disable-update.patch # Disable update and auto update functionality
+    ./add_list_option.patch # Add `list` option to be more intuitive (`git worktree list`)
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+
+    cp wt $out/bin
+    wrapProgram $out/bin/wt --prefix PATH : ${
+      lib.makeBinPath [
+        git
+        jq
+      ]
+    }
+
+    installShellCompletion --zsh completions/_wt_completion
+    installShellCompletion --bash completions/wt_completion
+    installShellCompletion --fish completions/wt.fish
+  '';
+
+  meta = with lib; {
```

Comment:
Avoid `with lib;`

Reply:
> What should I use instead then? I haven't found much documentation about this.

See FliegendeWurst's proposed changes

Reply:
Fixed in 7c2dda4abd2b5cef40276badf4f57dd391bc209f